src/core/server/server.api.md

```diff
@@ -1041,9 +1041,11 @@ export interface LegacyAPICaller {
     (endpoint: 'cat.help', params: CatHelpParams, options?: LegacyCallAPIOptions): ReturnType<Client['cat']['help']>;
     // (undocumented)
     (endpoint: 'cat.indices', params: CatIndicesParams, options?: LegacyCallAPIOptions): ReturnType<Client['cat']['indices']>;
-    // (undocumented)
+    // @deprecated (undocumented)
```

Comment:
This is a useful annotation, but it looks like it was maybe added manually? The comment at the top of the file implies this should be generated. Not a problem, but it means the deprecation notice might be lost upon regeneration.